CS-549.py

```diff
@@ -0,0 +1,47 @@
+from time import sleep
+import sys
+import requests
+import json
+
+# Accept arguments for auth and domain to work
+accessToken = sys.argv[1]
+domain = sys.argv[2]
+
+pageApiBase = "https://api.hubapi.com/cospages/v1/pages"
+
+# Get all pages on X subdomain 
+getPagesQueryString = (f"access_token={accessToken}&limit=2&state__in=PUBLISHED&state__in=PUBLISHED_AB&state__in=PUBLISHED_OR_SCHEDULED&subcategory__eq=site_page")
+
+getSitePages = requests.get(f"{pageApiBase}/list/{domain}", params = getPagesQueryString)
+getSitePagesObjects = getSitePages.json()["objects"]
+
+getSitePagesObjectsCount = len(getSitePagesObjects)
+print (f"Hold your butts! Unpublishing and archiving {getSitePagesObjectsCount} website pages :butt_holdings:")
+
+# For each page on X subdomain
+for sitePageObject in getSitePagesObjects: 
+    
+   # Save their {slug} and {id}
+   slug = sitePageObject["slug"]
+   id = sitePageObject["id"]
```

Comment:
Sorry, I think `id` is a saved variable here. Lets name it `pageId` or something instead and update every place we used `id`